web/app/mu-plugins/ppj/acf-json/group_5a1c011cdc0e7.json

```diff
@@ -74,9 +74,13 @@
                                         "id": ""
                                     },
                                     "choices": {
-                                        "one": "blue",
+                                        "one": "light blue",
                                         "two": "pink",
-                                        "three": "green"
+                                        "three": "mint",
+                                        "four: blue": "four: blue",
+                                        "five: red": "five: red",
+                                        "six: turquoise": "six: turquoise",
+                                        "seven: orange": "seven: orange"
```

Comment:
@jsmcgd This looks wrong. The keys and values for four, five, six & seven are the same. Shouldn't this be: `"four": "blue"` and so on?

ACF is weird about key/value pairs in select lists. You probably need to reformat the options in the GUI with a space before & after the colon:

`four : blue` instead of `four: blue`

Reply:
Yeah this is very wrong. Not sure what happened here.